### ISSUE TITLE
PV detach from shutdown node

### ISSUE THREAD

**Issue by lavalex** [kubernetes-sigs/nfs-subdir-external-provisioner#359]
Hello,

When node goes down, pod is deployed on another node, but NFS mount stays "attached" to the failed node. Is it expected to happen?

Thanks.

**Comment by niranjandarshann** [comment]
@lavalex  Yes, it’s expected that NFS mounts stay attached when a node fails.

**Comment by niranjandarshann** [comment]
If a node goes down unexpectedly, Kubernetes does not immediately clean up or forcefully unmount the NFS volume.
The PV remains "attached" to the failed node until Kubernetes detects the failure.

**Comment by niranjandarshann** [comment]
One Approach that is common check   persistentVolumeReclaimPolicy in pv.yaml and check it should be Retain i think this may fix it. As this approach is actively taken and work for majority of the people.
